composer.json

```diff
@@ -15,12 +14,17 @@
 
     ],
     "require": {
-        "php": ">=5.3.0",
+        "php": ">=5.6",
```

Comment:
I don't believe any of my changes in the actual SDK code that requires 5.6. However, I believe the PHPUnit tests I added do. Perhaps that requirement would be better served w/ a require-dev. Although, Paysafe may wish to consider dropping official support for 5.3 since PHP doesn't even release security updates for that version anymore.